src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const currentStateClone = { ...state };
```

Comment:
The current state is cloned using a shallow copy. If the state contains nested objects, changes to these nested objects will affect the original state and all clones. Consider using a deep copy method if nested objects are present in the state.